opnode/test/system_test.go

```diff
@@ -50,9 +46,18 @@ func defaultSystemConfig(t *testing.T) SystemConfig {
 			l2OutputHDPath:     10000000,
 			bssHDPath:          10000000,
 		},
+		DepositCFG: DepositContractConfig{
+			FinalizationPeriod: big.NewInt(60 * 60 * 24),
```

Comment:
Wouldn't you want to set this to a smaller value so that you could test withdrawals?

Reply:
Probably. I can edit it when I actually add withdrawal tests. I'm not sure how high a priority it is to be able to hand that off to auditors.

Reply:
Fair point. We can change when we add withdrawal tests.